n_queen_problem/NQueenProblem.java

```diff
@@ -0,0 +1,53 @@
+import java.util.*;
+
+public class NQueenProblem { 
+    private static void placeQueenOnBoard(int Qi, int[] board) {
+        int n = board.length;
+        //base case
+        if (Qi == n) {
+            System.out.println(Arrays.toString(board));
```

Comment:
What you can do is instead of `void` make return type `string` that would help you display in `main()`